keps/sig-node/127-user-namespaces/README.md

```diff
@@ -72,7 +75,7 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 ## Summary
 
-This KEP adds support to use user-namespaces in pods.
+This KEP adds support to use user-namespaces in stateless pods.
```

Comment:
Does this mean volumes are off the table or just punted to another KEP which hasn't been written?  Will it be written?  When and by whom?

Reply:
Yes, @giuseppe and me will work on stateful workloads too, after this work progresses (will talk with the community when we feel confident to move with stateful pods later). Probably as part of another KEP, to simplify the discussion here.